src/openrct2-ui/drawing/engines/opengl/DrawRectShader.cpp

```diff
@@ -27,11 +27,15 @@ constexpr VDStruct VertexData[4] = {
     { 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f, 1.0f, 1.0f, 1.0f },
 };
 
+constexpr const size_t InitialInstancesBufferSize = 32768;
+
 DrawRectShader::DrawRectShader()
     : OpenGLShaderProgram("drawrect")
```

Comment:
Can you move assignment to initialization list?

Reply:
Yeah, that makes sense, will do